pre-piksi-multi-launch-updates.md

```diff
@@ -0,0 +1,222 @@
+### SBP changes before shipping Piksi Multi
+
+There are a few changes we would like to make in the SBP protocol. These changes
+will be much more painful to make after we ship and customers are relying on
+the protocol to be largely static, so we'd like to come to a consensus on
+these and implement them before shipping Piksi Multi.
+
+These changes attempt to accomplish a few things:
+- Make messages more bandwidth efficient
+- Make user side logic of using messages more obvious
+- Remove state from the protocol, where possible
+
+### 1. Send solution messages at all times, regardless of fix status
+
+- Affects the following messages:
+   - GPS time
+   - DOPS (to add)
+   - Pos ECEF
+   - Pos LLH
+   - Baseline ECEF
+   - Baseline NED
+   - Vel ECEF
+   - Vel NED
+   - Baseline heading 
+- Update flags 
+   - Describe "fix mode" portion of flags field to be for each individual message only
+   - in all messages, 0 corresponds to "invalid"
+   - RTK Float and RTK Fixed flags have same numerical value across different messages where relevant
+   - No separate flags for simulation mode
+   - Only a minor release without a change to MSG_ID can be used to assign a meaning to reserved flags
+- Flags detail
+    - POS flags:
+       - Bits 2-0
+         - 000 Invalid
```

Comment:
Yeah, an actual float value (or clipped log) would be awesome.  Even better, return the full covariance matrix!  Though I imagine that would get resistance as it would involve sending 9 float values which would bloat these messages.  Assuming we stick to one number we'll have to think a bit about what exactly the "uncertainty" around an estimate is.  In otherwords, how do we summarize a full covariance matrix with a single number?  Probably we'd want to return the largest eigen value of the covariance to be conservatve ... but there are other options.